packages/orbit-components/src/Tile/components/TileContent/index.tsx

```diff
@@ -1,8 +1,5 @@
 import * as React from "react";
-import styled, { css } from "styled-components";
-
-import mq from "../../../utils/mediaQuery";
-import defaultTheme from "../../../defaultTheme";
+import clsx from "clsx";
```

Comment:
just 💅🏻 , we use it everywhere as import cx from "clsx" :)

```suggestion
import cx from "clsx";
```

Reply:
adjusted